backpack/core/derivatives/adaptive_avg_pool_nd.py

```diff
@@ -27,7 +28,11 @@ def check_parameters(
         Raises:
             NotImplementedError: if the given shapes do not match
         """
-        if ("cuda" in str(module.input0.device)) and (self.N == 3):
+        if (
+            TORCH_VERSION_HIGHER_THAN_1_9_1 is False
```

Comment:
the name suggests `>1.9.1`, although `>1.9.0` is meant, I guess.

Reply:
I'd like to express `torch >= 1.9.0`
Now: `TORCH_VERSION_HIGHER_THAN_1_9_1`
Other ideas:
- `TORCH_VERSION_GREATER_THAN_1_9_1`
- `TORCH_VERSION_AT_LEAST_1_9_1`

Is any of these versions better to understand than the others?

Reply:
I choose the last alternative `TORCH_VERSION_AT_LEAST_1_9_1`. Feel free to vote for a different one or to insist on something else entirely like for example `TORCH_VERSION_HIGHER_THAN_1_9_0`.

Reply:
Works. Resolved.